Y/Yosys/build_tarballs.jl

```diff
@@ -14,20 +14,25 @@ dependencies = [
     Dependency("boost_jll"; compat="=1.76.0"), # max gcc7
     Dependency("Readline_jll"),
     Dependency("Tcl_jll"),
-    Dependency("Zlib_jll"; compat="~1.2.11"),
-    Dependency("Libffi_jll"; compat="~3.2.2"),
+    Dependency("Zlib_jll"; compat="1.2.12"),
+    Dependency("Libffi_jll"; compat="3.2.2"),
```

Comment:
Libffi _*must*_ stay with
```suggestion
    Dependency("Libffi_jll"; compat="~3.2.2"),
```
see #3024.  Compat on Zlib is fairly useless

Reply:
Thanks! I've got that issue bookmarked now :)